iOSNativeTemplate/iOSNativeTemplate/AppDelegate.m

```diff
@@ -26,17 +26,9 @@
 #import "AppDelegate.h"
 #import "InitialViewController.h"
 #import "RootViewController.h"
-#import <SalesforceSDKCore/SFSDKAppConfig.h>
-#import <SalesforceSDKCore/SFPushNotificationManager.h>
-#import <SalesforceSDKCore/SFDefaultUserManagementViewController.h>
-#import <SalesforceSDKCore/SalesforceSDKManager.h>
-#import <SalesforceSDKCore/SFUserAccountManager.h>
 #import <MobileSync/MobileSyncSDKManager.h>
```

Comment:
Can we update for MobileSync and any of our other frameworks (Common, Analytics, SmartStore, ReactNative, Hybrid and FileLogger) if they're used?

Reply:
OK, yeah. I'll update those as well.